.github/workflows/docs_build.yml

```diff
@@ -0,0 +1,75 @@
+name: Docs Build
+on:
+  workflow_dispatch:
+    inputs:
+      version: { type: string, required: false, description: The version to build docs for, used as git tag and pypi version.  If not specified then use master and wheel from the last successful build.}
+      latest: { type: boolean, required: false, description: Alias this version as the 'latest' stable docs.}
```

Comment:
There should be some guidance on when this should be used.

Ideally, it should be inferred based on the latest version on PyPi. For example, you could allow `version` to take the special value `latest`, which downloads the latest wheel.

Reply:
I've added guidance.  I do need to know the version/tag.  I could figure it out from the wheel I get installed, but that's a little complicated.